ajax/libs/bootstrap-iconpicker/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "bootstrap-iconpicker",
+  "version": "1.8.2",
+  "filename": "js/bootstrap-iconpicker.min.js",
+  "description": "A simple iconpicker for Bootstrap 3.x",
+  "homepage": "https://victor-valencia.github.io/bootstrap-iconpicker/",
+  "keywords": [
+    "bootstrap",
+    "iconpicker",
+    "icon-fonts",
+    "icons",
+    "css",
+    "javascript",
+    "jquery"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/victor-valencia/bootstrap-iconpicker.git",
+    "fileMap": [
+      {
+        "basePath": "bootstrap-iconpicker",
+        "files": [
+          "**/*"
+        ]
+      }
+    ]
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/victor-valencia/bootstrap-iconpicker.git"
+  },
+  "license": "MIT",
+  "author": "Victor Valencia Rico @victor-valencia"
```

Comment:
Thanks for the contribution! @victor-valencia 
Can you modify this to 
```
"author": [
  "name": "Victor Valencia Rico",
  "email": "valencia_vik@hotmail.com.com"
]
```

like the package.json in your repo.